Updated advice on outgoing email

jtuliani opened the pull request:
Old link at https://blogs.msdn.microsoft.com/mast/2016/04/04/sending-e-mail-from-azure-compute-resource-to-external-domains/ is out of date, it points to a new link https://blogs.msdn.microsoft.com/mast/2017/11/15/enhanced-azure-security-for-sending-emails-november-2017-update/. Updated FAQ accordingly.

azurecla commented:
Hi __@murdockcrc__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

tysonn commented:
@chackdan - Chacko, could you review this public PR and submit an update to address the broken link? The target article does not exist and does not have a redirect.

tysonn commented:
@ChackDan - Second notice, please review and submit an update

ChackDan commented:
Pull # 24236 submitted

tysonn commented:
@murdockcrc  - Thanks for pointing this out, author will fix in another PR.

Commit:
Updated advice on outgoing email

Old link at https://blogs.msdn.microsoft.com/mast/2016/04/04/sending-e-mail-from-azure-compute-resource-to-external-domains/ is out of date, it points to a new link https://blogs.msdn.microsoft.com/mast/2017/11/15/enhanced-azure-security-for-sending-emails-november-2017-update/. Updated FAQ accordingly.
